src/pattern-library/components/Library.tsx

```diff
@@ -0,0 +1,386 @@
+import classnames from 'classnames';
```

Comment:
Nothing new is happening in this module. It's been converted to TSX (types added and exported), I've collapsed some internal components that are only used once, and added components from `LibraryNext`.